From 0dbdb3a07e3a184fd2a1b046664ecab3c4d6c06f Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 22 Dec 2011 09:37:32 +0800
Subject: [PATCH 0795/2508] ENGR00170819-3 usb-gadget: add usb audio support
 at imx6 defconfig

Add usb audio gadget support at imx6 defconfig

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 5fc5463..e1c9573 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1716,6 +1716,7 @@ CONFIG_USB_ARC=y
 CONFIG_USB_GADGET_DUALSPEED=y
 # CONFIG_USB_ZERO is not set
 # CONFIG_USB_AUDIO is not set
+CONFIG_USB_AUDIO=m
 CONFIG_USB_ETH=m
 CONFIG_USB_ETH_RNDIS=y
 # CONFIG_USB_ETH_EEM is not set
-- 
1.7.7.4

